packages/cli/src/commands/start.ts

```diff
@@ -192,8 +192,6 @@ export class Start extends BaseCommand {
 	}
 
 	async initOrchestration() {
-		if (config.getEnv('executions.mode') !== 'queue') return;
-
```

Comment:
This early return was preventing `OrchestrationService.init` from setting the instance type as `leader` in single-main setup.